PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Situation
+<!-- Describe the background or context leading to this change. Include why this work is needed (e.g., a specific problem, user need, or opportunity). What is the current state or issue that prompted this PR? -->
+
+# Task
+<!-- Explain the goal or objective of this change. What specifically needs to be achieved to resolve the situation? Clearly define the scope of the task at hand. -->
+
+# Action
+<!-- Summarize the key steps or decisions taken to accomplish the task. Include what changes were made in the codebase, architecture, or process. What actions were implemented to address the task? -->
+
```

Comment:
Should we add a `testing` section?

# Testing
Outline the tests conducted to ensure the changes work as expected. Include unit tests, integration tests, and manual tests.

Reply:
I added a new section for "Testing". The comment was auto-filled by co-pilot.